cumulusci/cumulusci.yml

```diff
@@ -151,6 +151,10 @@ tasks:
         description: Publishes a release to the mrbelvedere web installer
         class_path: cumulusci.tasks.mrbelvedere.MrbelvederePublish
         group: Release Management
+    metadeploy_publish:
```

Comment:
I wonder if we should leave this out of the global cumulusci.yml for now.  It really needs to be configured in the project to work at all right?  We might be able to have a default one that uses the new install_prod_no_config flow in a future release, but it seems like we might want to keep this project specific for now.